client/service/src/builder.rs

```diff
@@ -87,7 +92,21 @@ impl ServiceMetrics {
 			node_roles: register(Gauge::new(
 				"node_roles", "The roles the node is running as",
 			)?, registry)?,
-
+			database_cache: register(Gauge::new(
+				"database_cache", "Database cache size",
```

Comment:
Check out the Prometheus metrics naming guide: https://prometheus.io/docs/practices/naming/ . These should have `_bytes` added onto the end of the name and `in bytes` added onto the help text.